### ISSUE TITLE
Fix `at-rule-prelude-no-invalid` false positives for `@import`

### ISSUE THREAD

**Issue by coderaiser** [stylelint/stylelint#9472]
### What minimal example or steps are needed to reproduce the bug?

what is wrong with this css?

```css
@import url('./highlight.css');
@import url('../fontcustom/fontcustom.css');
```

I get two errors:

```
 1:9   error   Invalid prelude "url('./highlight.css')" for at-rule "@import"               at-rule-prelude-no-invalid (stylelint) 
 2:9   error   Invalid prelude "url('../fontcustom/fontcustom.css')" for at-rule "@import"  at-rule-prelude-no-invalid (stylelint) 
 1:13  error   Expected no quotes around "url" function argument                            function-url-quotes (stylelint)        
 2:13  error   Expected no quotes around "url" function argument                            function-url-quotes (stylelint) 
```

With this

```css
@import './highlight.css';
@import '../fontcustom/fontcustom.css';
```

I have:

```
 1:9  error   Invalid prelude "'./highlight.css'" for at-rule "@import"                              at-rule-prelude-no-invalid (stylelint) 
 2:9  error   Invalid prelude "'../fontcustom/fontcustom.css'" for at-rule "@import"                 at-rule-prelude-no-invalid (stylelint) 
 1:9  error   Expected "'./highlight.css'" to be "url('./highlight.css')"                            import-notation (stylelint)            
 2:9  error   Expected "'../fontcustom/fontcustom.css'" to be "url('../fontcustom/fontcustom.css')"  import-notation (stylelint)
```

Which one is correct and should I use, looks like bug in `at-rule-prelude-no-invalid`

### What minimal configuration is needed to reproduce the bug?

```json
{
    "extends": ["stylelint-config-standard"],
    "rules": {
        "function-url-quotes": "always",
    },
    "plugins": [
        "@stylistic/stylelint-plugin"
    ]
}

```

### How did you run Stylelint?

api

### Which Stylelint-related dependencies are you using?

latest

### What did you expect to happen?

no warnings

### What actually happened?

see errors

**Comment by jeddy3** [comment]
@coderaiser Thanks for the report.

I can reproduce it using [the demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBLAtgBwgJwC4AIBXbAGwAoByAOgHoALFAczpKbtyrAGcuKBKANwAdAHapMOAsXLVaAMwgjcYQl1wQ0NBUpVqNnHvwEgANCDkoSMAHIBDNHEQwAHvYxWDXU+EUXGCEBcsPHxYOVtCEgJgUXx8F1wYESgueHwAbQo1AE8rViUAWkgRPwK1W2TbbCgKAF0TUQBfYRFvYr8AMRw0W1wAnLyUJU5fJio0ACsvM1gML0QYkTihEAGYfNwVtJWSXpg1FYal-BW1jaLRxjLcCqgqqC2TkF3Eg5Am0RBGoA).

The [prelude is valid](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@import) and shouldn't have been flagged.

The issue may have been triggered upstream by this change in the [`css-syntax-patches-for-csstree`](https://www.npmjs.com/package/@csstools/css-syntax-patches-for-csstree) package:
- https://github.com/csstools/postcss-plugins/pull/1899/changes

**Comment by romainmenke** [comment]
Thank you for reporting this (and pinging me).
This has been resolved.
The [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBLAtgBwgJwC4AIBXbAGwAoByAOgHoALFAczpKbtyrAGcuKBKANwAdAHapMOAsXLVaAMwgjcYQl1wQ0NBUpVqNnHvwEgANCDkoSMAHIBDNHEQwAHvYxWDXU+EUXGCEBcsPHxYOVtCEgJgUXx8F1wYESgueHwAbQo1AE8rViUAWkgRPwK1W2TbbCgKAF0TUQBfYRFvYr8AMRw0W1wAnLyUJU5fJio0ACsvM1gML0QYkTihEAGYfNwVtJWSXpg1FYal-BW1jaLRxjLcCqgqqC2TkF3Eg5Am0RBGoA) is already showing the updated version.

Spec fix: https://github.com/w3c/csswg-drafts/pull/14419
Patches fix: https://github.com/csstools/postcss-plugins/commit/d2536edda6ad1e4f6fe9859a900035dd9f9cad0c

**Comment by romainmenke** [comment]
@jeddy3 I missed the `function-url-quotes (stylelint)` errors.
Is that something we should still look into?

**Comment by jeddy3** [comment]
@romainmenke Thank you very much for the quick fix upstream, and for opening a fix to the spec further upstream!

> I missed the `function-url-quotes (stylelint)` errors. Is that something we should still look into?

I can't replicate this with [the demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBLAtgBwgJwC4AIBXbAGwAoAdEAMwggDowBnJqgSgG4KA7VTHAsXIAjAIbZGLTiAA0NFCRgA5UWjiIYAD1UZFkprPARu1FAHMEILVjz5Y1UYRIFgPfPi24Y3KE3j4AbQByJlwAT0USFG5cAFpIE3NY0NEfcSgggF0ZN3wqbCcYVgR8V253CoqqakJuMFwUY1ihWIBHQggvYv8qURIAd1Ew4pzyioBfUfGubkME0zMAMRw0UVxLUIiYKJjGYwX6NAArAzlYDANEMvcqTcjo3CoekBI1osfZXNvw+5j4-aSKTS2CgTzyLzeoSoPHGPBA4yAA), so assuming it's okay. (Our standard config [includes](https://github.com/stylelint/stylelint-config-standard/blob/8fae02f35eca87a9b22397936062d88b3df63292/index.js#L72) `function-url-quotes: 'always'`). @coderaiser You're welcome to open another issue if it isn't, as it's unlikely to be related to the csstree patches change.
